Natasha_Telesheva_HW1.cpp

```diff
@@ -0,0 +1,42 @@
+
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main()
+{
+    const int kConst1{ 2 };
+    const int kConst2{ 4 };
+
+    double a{}, b{}, c{}, d{}, x1{}, x2{};
+    cout <<" Please enter numbers: a,b,c in range[2..255]\n";
+    cin >> a;
+    if (a == 0) {
+        cout << "The equation is not quadratic" << endl;
+        return 0;
+    }
+    cin>> b >> c;
+    if ((a < 2) || (b< 2) || (c< 2) || (a  > 255) || (b>255) ||(c>255) ){
+        cout << "You entered an incorrect number, try entering the number again";
+        return 0;
+    }
+    cout << "Your numbers:" << a << " "<< b << " " << " " << c << endl;;
+
+    d = b * b - kConst2 * a * c;
```

Comment:
например я бы рекомендовал тут обьявлять переменную `d`
```suggestion
   const long double d  { b * b - kConst2 * a * c };
```